docusaurus/docs/tutorial/01.md

```diff
@@ -3,213 +3,442 @@ id: 01
 title: 01. Creating Your First Page
 sidebar_label: 01. Creating Your First Page
 ---
-In this section, we will try to create a new page on our Treats project.  
-There are two main steps in this tutorial:  
-1. Creating your page and style it
-2. Register your page to application route
+__Treats using React as it's core. If you don't get a grip on React concept, we suggest you to learn more about [React][React] before following this tutorial.__
 
+In this section, we will try to create a simple `todo-apps` using Treats.
+Install __`create-treats-app`__ packages and run the command.
 
-Before we start, let's assume we want to create a page named __`mypage`__.  
-__`Please note that this name is an example and you may change it with anything.`__
+```
+// yarn
+~$ yarn global add create-treats-app
+
+// npm
+~$ npm install -g create-treats-app
+
+~$ create-treats-app
+```
+
+A prompt asking project's name, version, description, and usage of Typescript will be shown. Please fill it all or just enter if you want it to be filled with the default values. Before we start, let's assume we want to create a page named `todo`.  
+
+__Please note that this name is an example and you may change it with anything.__
 
 Now, let's get started:
 
-1. Create new folder named __`mypage`__, containing __`index.js`__ and __`mypage.css`__. Your folder should look like this:  
+## Creating Component
+Before creating our page and to improve our understanding in creating components using Treats, let's make our list in separate component.  
```

Comment:
instead of saying this `Before creating our page and to improve our understanding`, we can just say that the first step is creating list component.